documentation/polymer-templates/tutorial-template-event-handlers.asciidoc

```diff
@@ -1,12 +1,12 @@
 ---
-title: Handling User Events in a PolymerTemplate
-order: 4
+title: PolymerTemplate, Handling User Events
+order: 8
 layout: page
 ---
 
 ifdef::env-github[:outfilesuffix: .asciidoc]
 
-= Handling User Events in a PolymerTemplate
+= PolymerTemplate, Handling User Events
 
 == Defining Client-side Event Handlers
```

Comment:
I wonder if a note should be added for LitElement users that the event handlers are recommended to be added in the Java side? This comment applies to all `PolymerTemplate, ...` page.. What to do if you are using LitElement? It may now look like there's a solution for Polymer, but not for LitElement.

Reply:
I'll leave it as is for now. There is an example already in the "Binding Template Components" section. All sections with "PolymerTemplate" syntax are kinda advanced.